{"name": "Проверка ответа при запросе на получение списка заказов неаутентифицированного пользователя", "status": "passed", "start": 1725975029933, "stop": 1725975030295, "uuid": "17f68369-e872-4290-88a3-5cfadad80a7a", "historyId": "d932a420574a8af1ba53e8754c88e341", "testCaseId": "d932a420574a8af1ba53e8754c88e341", "fullName": "tests.test_get_user_orders.TestGetOrders#test_get_orders_unauthenticated_user_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "DESKTOP-82VEM3U"}, {"name": "thread", "value": "13288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_orders"}]}